src/webextension/locales/en-US/list.ftl

```diff
@@ -71,7 +71,7 @@ homepage-greeting =
 
 homepage-linkaccount-title = Add Serious Security & Convenience
 homepage-linkaccount-description =
-    Creating a Firefox account - or adding { product-title } to an existing account - protects your logins
+    Creating a Firefox account – or adding { product-title } to an existing account – protects your logins
```

Comment:
One more micro-nit: I think em dashes usually don't have space around them, but feel free to double-check with Ryan.

Reply:
The copy we got had spaces around them, and @changecourse thinks they should have spaces around them, too.

Reply:
I looked it up and I guess I use Chicago style in my writing, whereas this is just AP style. :)

Reply:
> this is just AP style

![more-you-know](https://user-images.githubusercontent.com/49511/33744948-3abeeee6-db71-11e7-8c7f-dcf3ba8dd658.gif)

We use coast-based style guides around here, none of that mid-west stuff. 😏 🤠